prediction_market_agent/agents/omen_cleaner_agent/deploy.py

```diff
@@ -55,12 +55,11 @@ def clean(self) -> None:
     @observe()
     def resolve_finalized_markets(self, api_keys: APIKeys) -> list[HexAddress]:
         logger.info("Resolving finalized markets.")
-        # Delay by 24 hours to give a chance for the market creator to be resolve it.
         finalized_unresolved_markets = [
             market
             for market in OmenSubgraphHandler().get_omen_binary_markets(
                 limit=None,
-                finalized_before=utcnow() - timedelta(hours=24),
+                finalized_before=utcnow(),
```

Comment:
Before I thought it was a good idea to give some time for market-creator to resolve it by himself, but now I think otherwise because it stabbed me in the back 😄 

Why wait for market-creator if we can resolve it right away and give people their money?